### DISCUSSION TITLE
复制链接功能无法使用

### DISCUSSION THREAD

**Discussion by MC-dusk** [the1812/Bilibili-Evolved#1329] [综合讨论 / General]
（我理解的“复制链接”功能指“复制评论/动态对应的url，比如“https://t.bilibili.com/111 ”或“https://www.bilibili.com/video/av123#reply456 ”）

如题，无论是“复制评论链接”还是“复制动态链接”，开启功能后菜单有按钮，但按下后无反应，点粘贴什么也没有。
如果之前剪贴板有内容，比如已经复制了“abc”，按下“复制链接”后再粘贴，粘贴不出东西，也就是说剪贴板被重写了，但没写入链接。

我看issue没人提这个问题，应该不是个bug，是我没配置好吗？

火狐83.0
暴力猴v2.12.8
脚本 Stable v1.11.5 内容包: 7a73519

**Comment by Geeyun-JY3** [comment]
这边 Chrome + Tampermonkey 点击 `复制评论链接` 后评论的更多菜单确实不会消失，但是是复制成功的：#1196

**Reply by MC-dusk** [reply]
我用Tampermonkey也是这样——菜单不会消失，但复制成功，可以粘贴
那么我报issue了
